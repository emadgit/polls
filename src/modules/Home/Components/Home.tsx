import React from 'react'
import Typography from '@material-ui/core/Typography';

export default function Home() {
  return (
    <div className="viewSection">
      <Typography component="h2" variant="display2" className="whiteColor" gutterBottom>
          Welcome to Pools App!
        </Typography>
        <Typography component="p" variant="body2" className="whiteColor">
          Here you can view polls and vote in them ;)
        </Typography>
    </div>
  )
}
