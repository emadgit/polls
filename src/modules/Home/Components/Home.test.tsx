import * as React from "react";
import { shallow } from "enzyme";
import Home from "./Home";
describe("Home Component", () => {
  it("should render correctly", () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
