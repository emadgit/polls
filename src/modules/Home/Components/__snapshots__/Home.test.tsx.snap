// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render correctly 1`] = `
<div
  className="viewSection"
>
  <WithStyles(Typography)
    className="whiteColor"
    component="h2"
    gutterBottom={true}
    variant="display2"
  >
    Welcome to Pools App!
  </WithStyles(Typography)>
  <WithStyles(Typography)
    className="whiteColor"
    component="p"
    variant="body2"
  >
    Here you can view polls and vote in them ;)
  </WithStyles(Typography)>
</div>
`;
